cliconfig/credentials/default_store_unix.go

```diff
@@ -2,4 +2,4 @@
 
 package credentials
 
-const defaultCredentialsStore = ""
+const defaultCredentialsStore = "secretservice"
```

Comment:
But what is really using under the hood is a secret service  (.service),hence the name